edx_reverification_block/xblock/reverification_block.py

```diff
@@ -107,6 +107,8 @@ def student_view(self, context=None):
 
         This will render the url to display in lms along with marketing text.
         """
+        precedence_statuses = ["submitted", "approved", "denied"]
```

Comment:
Please add some comment why we need precedence for statuses, also what are these statuses since these statuses aren't defined directly here?